addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -134,7 +134,7 @@ def _make_request(self, url, params=False):
         if 'error' in response:
             message = _('The url that this service requested returned an error. The url it tried to contact was %(url)s. %(error_message)s', url=url, error_message=response['error']['message'])
             if response['error']['code'] == 404:
-                message = _('The url that this service tried to contact does not exist. The url was %r', url)
+                message = _('The url that this service tried to contact does not exist. The url was “%s”', url)
```

Comment:
I think this sounds better:
```suggestion
                message = _('The url that this service tried to contact does not exist. The url in question is “%s”', url)
```

That being said, the error messages above this all have the same `was` wording that sounds a little strange to me so :shrug: probably doesn't matter